package.json

```
@@ -51,15 +50,38 @@
     "url": "https://github.com/mlibrary/lib.umich.edu"
   },
   "devDependencies": {
+    "@eslint/compat": "^1.0.1",
+    "@eslint/js": "^9.2.0",
+    "@stylistic/eslint-plugin": "^2.1.0",
     "bottleneck": "^2.19.5",
     "entities": "^4.5.0",
-    "eslint": "^8.57.0",
-    "eslint-config-standard": "^17.1.0",
-    "eslint-config-standard-jsx": "^11.0.0",
-    "eslint-config-standard-react": "^13.0.0",
+    "eslint": "^9.2.0",
     "eslint-plugin-react": "^7.34.1",
     "fetch-retry": "^6.0.0",
     "gatsby-plugin-netlify": "^5.1.1",
-    "gatsby-plugin-remove-serviceworker": "^1.0.0"
+    "gatsby-plugin-remove-serviceworker": "^1.0.0",
+    "globals": "^15.2.0"
+  },
+  "overrides": {
+    "eslint-plugin-react": {
+      "eslint": "latest"
+    },
+    "react-scripts": {
+      "@svgr/webpack": "latest",
+      "babel-preset-react-app": {
+        "@babel/plugin-proposal-class-properties": "npm:@babel/plugin-transform-class-properties",
+        "@babel/plugin-proposal-nullish-coalescing-operator": "npm:@babel/plugin-transform-nullish-coalescing-operator",
+        "@babel/plugin-proposal-numeric-separator": "npm:@babel/plugin-transform-numeric-separator",
+        "@babel/plugin-proposal-optional-chaining": "npm:@babel/plugin-transform-optional-chaining",
+        "@babel/plugin-proposal-private-methods": "npm:@babel/plugin-transform-private-methods"
+      },
+      "css-minimizer-webpack-plugin": "latest",
+      "jest": "latest",
+      "magic-string": "latest",
+      "resolve-url-loader": "latest",
+      "source-map-loader": "latest",
+      "typescript": "latest",
+      "workbox-webpack-plugin": "latest"
+    }
   }
-}
+}
```

```suggestion
}

```
Silly new line error.